packages/error-boundary-fallback/src/components/ErrorBoundary/index.tsx

```diff
@@ -1,63 +1,30 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { Result, Button } from 'antd';
-import * as Sentry from '@sentry/react';
+import { PAGE_TITLE, PAGE_SUB_TITLE, BUTTON_TITLE } from '../../lang';
 
 interface Props {
-  children: unknown;
-  dsn: string;
   homeUrl?: string;
 }
 
-interface State {
-  hasError: boolean;
-}
-
-class ErrorBoundary extends Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      hasError: false,
-    };
-  }
-
-  componentDidMount() {
-    const { dsn } = this.props;
-    if (dsn && dsn !== '') {
-      Sentry.init({ dsn: dsn });
-    }
-  }
-
-  componentDidCatch(err: unknown) {
-    this.setState({ hasError: true });
-    Sentry.captureException(err);
-  }
-
-  render() {
-    const { hasError } = this.state;
-    const { homeUrl, children } = this.props;
-    if (hasError) {
-      return (
-        <Result
-          status="error"
-          title="An Error Occurred"
-          subTitle="There has been an error. It’s been reported to the site administrators via email and
-        should be fixed shortly. Thanks for your patience."
-          extra={
-            <Button
-              id="backHome"
-              key="error"
-              onClick={() => (window.location.href = homeUrl ? homeUrl : '/')}
-              type="primary"
-            >
-              Back Home
-            </Button>
-          }
-        />
-      );
-    } else {
-      return children ? children : <></>;
-    }
-  }
-}
+const ErrorBoundary = (props: Props) => {
```

Comment:
Should this be error fallback instead of error boundary?

Reply:
Yes, makes sense.

Reply:
Only the component name or also the package name? currently we are using `error-boundary-fallback`

Reply:
the component name

Reply:
also immediate component's file name

Reply:
yeah I meant `error-boundary-fallback` instead of `ErrorBoundary`

Reply:
Changing `@opensrp/error-boundary-fallback` to `@opensrp/error-fallback` and all relevant declarations.

Reply:
the package naming is ok, what needs to change is the components filename and the component itself, since the package is exporting  erorBoundaryFallback and not errorBoundary